hub/src/main/java/com/hub/HubController.java

```diff
@@ -65,6 +59,7 @@ protected String findLeader() throws IOException {
                         .send(request, HttpResponse.BodyHandlers.ofString());
                 if(response.body().equals("true")){
                     leader = String.valueOf(id);
+                    sendLeader();
```

Comment:
Now that we are sending the leader as well, I would suggest we rename to `findAndSendLeader`